tests/unit/models/test_settings.py

```diff
@@ -0,0 +1,22 @@
+from runner_manager.models.settings import Settings
+from pytest import fixture
+
+
+def test_settings_default_values():
+    settings = Settings()
+    assert settings.name == "runner-manager"
+    assert settings.redis_om_url == "redis://localhost:6379/0"
+    assert settings.github_base_url == "http://localhost:4010"
```

Comment:
did I put that as default value? I'm surprised.

Reply:
I think without the @fixture it just picks up the value used by the settings passed in during the startup of application